### PR TITLE
Expand branding api

### PR DESCRIPTION
**Pull request by HowlingEverett**
Standardises the exports of the importable SVGs, pre-compiles them.

Also introduces the `ui-logo` module which is just a component that loads a particular logo and pads it (optionally)

Also adds syntax highlighting of demo code, and a doc page for the component that renders it and its demo source.

screen shot 2017-03-28 at 11 51 50 am

### CODE REVIEW

**Comment by lukebrooker** [comment]
@HowlingEverett Could you add a basic example to Experience Docs > Components? Even if you have a Example.md in `ui-logo` that you import with Experience Docs?

**Comment by lukebrooker** [comment]
LGTM 👍

**Comment by benoneal** [comment]
Rounded corners on code blocks, then 👍  when green.

**Comment by lukebrooker** [comment]
Just needs `navTitle: Logo` in the Frontmatter

File: packages/experience-docs/source/content/components/logo.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Logo Component
+path: /components/logo
```
Comment (lukebrooker) [inline packages/experience-docs/source/content/components/logo.md]: `navTitle: Logo`

**Review by lukebrooker — CHANGES_REQUESTED** [review]

**Review by lukebrooker — APPROVED** [review]

**Pull request closed without merging**